src/pages/Main.js

```diff
@@ -31,6 +32,11 @@ export default function Main(props) {
         <div></div>
       </section>
       <section className="timeline-section"></section>
+      <section className='footnote'>
+        <div>
+          <Footnote />
+        </div>
+      </section>
```

Comment:
You can use `<footer></footer>` instead of div, and you might not need the div outside `<Footnote />`